integreat_cms/cms/templates/regions/region_form.html

```diff
@@ -291,6 +294,63 @@ <h3 class="heading font-bold text-black">
                             </p>
                         </div>
                     {% endif %}
+                    {% if form.instance.id %}
+                        <div class="max-w-xs">
+                            <div class="flex flex-wrap justify-between">
+                                <label class="secondary m-0">
+                                    {% trans "Current total budget" %}:
+                                </label>
+                                <p>
+                                    {{ form.instance.summ_ai_budget|intcomma }}
+                                </p>
+                            </div>
+                            <div class="flex flex-wrap justify-between">
+                                <label class="secondary m-0">
+                                    {% trans "Already consumed" %}:
```

Comment:
Super nit: We call the variable _used, so I'd proposed to change this to "Already used", but not at all important and I'm fine either way :)

Reply:
Done, for deepl as well